addons/l10n_fr_pos_cert/static/tests/tours/ModifyOderTour.js

```diff
@@ -0,0 +1,51 @@
+/** @odoo-module */
+
+import * as Chrome from "@point_of_sale/../tests/tours/helpers/ChromeTourMethods";
+import * as ProductScreen from "@point_of_sale/../tests/tours/helpers/ProductScreenTourMethods";
+import * as Order from "@point_of_sale/../tests/tours/helpers/generic_components/OrderWidgetMethods";
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("ModifySavedUnpaidOrder", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            ProductScreen.confirmOpeningPopup(),
+            ProductScreen.clickHomeCategory(),
+            ProductScreen.addOrderline("product_a", "1", "10"),
+            ProductScreen.clickBackend(),
+            // Continue selling, the order will be saved
+            Chrome.openSession(),
```

Comment:
Instead of closing and opening again (this takes a lot of time). You can create a new order and go back to the original one. It will save it and have the same behavior without reloading the PoS